### ISSUE TITLE
Como codar no Linux sem Photoshop?

### ISSUE THREAD

**Issue by theandersonn** [woliveiras/FFAQ#9]

**Comment by cristofersousa** [comment]
Acho que o termo **codar** ficou meio estranho ainda mais para Linux, acredito que você queira se referir a que tipos de software gráficos adotar no Linux que possa substituir a familia Adobe.

[Inkspace - CorelDraw](http://wiki.softwarelivre.org/InkscapeBrasil/Tutoriais)
[Gimp - Photoshop](http://www.gimpbrasil.org/)

Abraços

**Comment by woliveiras** [comment]
Dei uma editada no titulo pra ver se fica melhor :)

**Comment by woliveiras** [comment]
Concordo com o @cristofersousa. Trabalho a dois anos e pouquinho como Front e não precisei de Photoshops nesse período.

Somente quando trabalhava em agência e fazia de tudo.

Também pode ser que a pessoa seja freela, ai acho que ela vai precisar de ferramentas desse tipo.

**Comment by DWoonD** [comment]
Além das ferramentas citadas, você pode utlilizar o Brackets da adobe, é um editor de texto que você consegue abrir o psd, e ele te da algumas coisas, como cor, espaçamento etc...

Ou ainda você pode pedir para a pessoa (partindo do princípio que você está recebendo o psd de alguém) subir o psd no zeplin e com isso você tmbm consegue extrair informações e cores, espaçamento etc.

**Comment by theandersonn** [comment]
@DWoonD A questão que eu ainda não me adaptei é justamente essa `precisar recortar imagens`.
Além do `brackets` sugerem outra ferramenta?
